website/docs/r/time_series_insights_environment.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Time Series Insights"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_time_series_insights_environment"
+description: |-
+  Manages an Azure Time Series Insights Environment.
+---
+
+# azurerm_time_series_insights_environment
+
+Manages an Azure Time Series Insights Environment.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "northeurope"
+}
+resource "azurerm_time_series_insights_environment" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "S1_1"
+  data_retention_time = "P30D"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Azure Time Series Insights Environment. Changing this forces a new resource to be created. Must be globally unique.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Azure Time Series Insights Environment.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `sku_name` - (Required) Specifies the SKU Name for this Time Series Insights Environment. It is string consisting of two parts separated by an underscore(\_).The fist part is the `name`, valid values include: `S1` and `S2`. The second part is the `capacity` (e.g. the number of deployed units of the `sku`), which must be a positive `integer` (e.g. `S1_1`).
+
+* `data_retention_time` - (Required) Specifies the ISO8601 timespan specifying the minimum number of days the environment's events will be available for query.
+
+* `storage_limited_exceeded_behavior` - (Optional) Specifies the behavior the Time Series Insights service should take when the environment's capacity has been exceeded. Valid values include `PauseIngress` and `PurgeOldData`.
+
+* `partition_key` - (Optional) The name of the event property which will be used to partition data.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
```

Comment:
Is the `tags` ForceNew as defined in the schema? If it is, we should change this description accordingly.

Reply:
My fault here! Tags now updates without recreating the resource